src/metabase/util.clj

```diff
@@ -174,7 +211,7 @@
   [^String s]
   (boolean (when (string? s)
              (ignore-exceptions
-               (->Timestamp s)))))
+               (->Timestamp s utc)))))
```

Comment:
why are we defaulting to utc here?

Reply:
The function returns `true` if the string is parseable as a date, so the results of the parse are discarded. `->Timestamp` needs a timezone to parse a string, so I just hard coded it to UTC